ThreeBotPackages/zerobot/admin/actors/notifications.py

```diff
@@ -1,5 +1,9 @@
 from Jumpscale import j
-from threesdk.InstallTools import Tools
+
+try:
+    from threesdk.InstallTools import Tools
+except:
+    from .core.InstallTools import Tools
```

Comment:
should be `from Jumpscale.core.InstallTools import Tools`
should be full import . is relative to install folder

Reply:
or even better why that, can't we just use `j.core.tools. .` ?

Reply:
done :+1:  thanks waleed